interpreter/interpreter.go

```diff
@@ -67,6 +69,7 @@ func (i *Interpreter) SetScope(scope context.Scope) {
 func (i *Interpreter) restart() error {
 	i.ctx.Restarts++
 	i.Debugger.Message(fmt.Sprintf("Restarted (%d) time", i.ctx.Restarts))
+	i.ctx.BackendRequest = nil
```

Comment:
Probably we need to reset request/response related pointers like:

- `i.ctx.Object`
- `i.ctx.BackendResponse`
- `i.ctx.Response`

Reply:
Good point.

I agree with you. Apparently, we don't need these nil checks 👍

Reply:
Done.